01_lists/map-reduce-filter-etc/CountLettersInEveryWord.py

```diff
@@ -0,0 +1,4 @@
+def CountLettersInEveryWord(list):
```

Comment:
Stick to the python naming conventions.
https://realpython.com/python-pep8/#naming-styles